src/NServiceBus.ContainerTests/NServiceBus.ContainerTests.csproj

```diff
@@ -13,23 +15,6 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" PrivateAssets="All" />
     <PackageReference Include="NUnit" Version="[3.12.0, 4.0.0)" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" PrivateAssets="All" />
-    <PackageReference Include="Particular.Packaging" Version="0.9.0" PrivateAssets="All" />
   </ItemGroup>
 
-  <PropertyGroup>
```

Comment:
Given we no longer ship them downstream (I assume we assume containers providing a service collection/provider adapter already conform) would it make sense to online those tests into the unit test project? It seems right now they are just testing if lightinject works

Reply:
yeah, maybe but there are some thoughts about running those tests against Microsofts DI adapter as that is probably going to be the most common container used in the future and we have no tests against that container regarding the assumptions core makes on the container.